webpack.config.min.js

```diff
@@ -3,21 +3,22 @@
 var webpack = require('webpack');
 
 module.exports = {
-    entry: './src/timekit.js',
-    devtool: 'source-map',
-    output: {
-        path: './dist',
-        filename: 'timekit-sdk.min.js',
-        libraryTarget: 'umd',
-        library: 'timekit'
-    },
-    plugins: [
-        new webpack.IgnorePlugin(/vertx/),
-        new webpack.optimize.UglifyJsPlugin({minimize: true}),
-        new webpack.optimize.OccurenceOrderPlugin(),
-        new webpack.optimize.DedupePlugin(),
-        new webpack.ProvidePlugin({
-          Promise: 'es6-promise-promise'
-        })
-    ]
+	mode: 'production',
+	entry: './src/timekit.js',
+	devtool: 'source-map',
+	output: {
+		path: __dirname + '/dist',
```

Comment:
This too

Reply:
This change is a part of webpack upgrade from 1 to 4